### ISSUE TITLE
Widget column selection general configuration

### ISSUE THREAD

**Issue by Tomansion** [debiai/DebiAI#183]
**Is your feature request related to a problem? Please describe.**
All widgets have to check for their columns, we could define, in a func or in conf, rules about columns unique values limit and type. The widget will only the compatible functions to be selected.

Maybe a parameter of the columns component modal ?
![image](https://github.com/debiai/debiai/assets/59999324/5dff2faf-6c36-486e-ac8c-024664b8298a)
Maybe ask a designer what would be the best way to give a hint on what column to use

TLDR : Widgets can set warning rules, commanded columns types or category, and unsupported columns.

**Comment by Tomansion** [comment]
Done with unfolding update
